src/MiniCover/HitServices/HitServiceWithTests.cs

```diff
@@ -1,105 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Reflection;
-using MiniCover.HitServices;
-using MiniCover.Utils;
-
-namespace MiniCover
-{
-    public static class HitServiceWithTests
```

Comment:
I think a single HitService with toggles to enable features would be better.
Soon new features will appear and we will start to duplicate code on both HitServices.